### PR TITLE
API-5152 Add 'get-customer'.

### PR DESCRIPTION
**Pull request by gluszczyk**

### CODE REVIEW

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 1]
```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
...
+
+| Parameter     | Required | Data type | Notes |
+| ------------- | -------- | --------- | ----- |
+| `customer_id` | Yes      | string    |       |
+
+#### Response
+
+|              |                                                                                                                                        |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`         | [Customer's](#customer) id.                                                                                                            |
```
Comment (bkubiak) [inline content/messaging/agent-chat-api/v3.2/index.mdx]: Descriptions of other methods' responses have also `type` column, e.g.: https://developers.livechatinc.com/docs/messaging/agent-chat-api/#get-chats-summary

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 2]
```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
 <Section>
 <Text>
 
+### Get Customer
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx]: Please, also remember about adding the new methods to the **Available methods** table in both Web and RTM :)

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.2/index.mdx]: + Update the changelog with the following info, please:
There's a new method **Get Customer** ([Web](/messaging/agent-chat-api/v3.2/#get-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-customer)) that returns info about a given Customer.

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 3]
```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
...
+
+Returns the info about the Customer with given id.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx]:
```suggestion
Returns the info about the Customer with a given `customer_id`.
```

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 4]
```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
...
+| Parameter     | Required | Data type | Notes |
+| ------------- | -------- | --------- | ----- |
+| `customer_id` | Yes      | string    |       |
+
+#### Response
+
+|              |                                                                                                                                        |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`         | [Customer's](#customer) id.                                                                                                            |
+| `type`       | `customer`                                                                                                                             |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx]: I suggest we link to the **customer object** here and remove the links in other fields (name, email, avatar, etc.) I know this pattern is copied from Get Customer in Customer API, but it seems like 6 links pointing to the same place is too much.

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 5]
```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
...
+| `email`      | [Customer's](#customer) email. Returned only if set.                                                                                   |
+| `avatar`     | [Customer's](#customer) avatar. Returned only if set.                                                                                  |
+| `created_at` | When [customer's](#customer) identity was created.                                                                                     |
+| `fields`     | Custom fields. Ephemeral. Returned only if set.                                                                                        |
+| `statistics` | Counters for started threads, opened pages, etc.                                                                                       |
+| `last_visit` | Geolocation and opened pages from [customer's](#customer) most recent online visit. Returned only if customer logged in at least once. |
+| `chat_ids`   | Ids of customer's chats. Returned only if customer had at least one chat.                                                              |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx]:
```suggestion
| `chat_ids`   | IDs of a customer's chats. Returned only if the customer had at least one chat.                                                              |
```

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.2/index.mdx]: I'm using "ID" throughout the whole document, so let's stick to one version.

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 6]
```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
...
+| `name`       | [Customer's](#customer) name. Returned only if set.                                                                                    |
+| `email`      | [Customer's](#customer) email. Returned only if set.                                                                                   |
+| `avatar`     | [Customer's](#customer) avatar. Returned only if set.                                                                                  |
+| `created_at` | When [customer's](#customer) identity was created.                                                                                     |
+| `fields`     | Custom fields. Ephemeral. Returned only if set.                                                                                        |
+| `statistics` | Counters for started threads, opened pages, etc.                                                                                       |
+| `last_visit` | Geolocation and opened pages from [customer's](#customer) most recent online visit. Returned only if customer logged in at least once. |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx]:
```suggestion
| `last_visit` | Geolocation and opened pages from the [customer's](#customer) most recent online visit. Returned only if customer logged in at least once. |
```

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 7]
```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
...
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`         | [Customer's](#customer) id.                                                                                                            |
+| `type`       | `customer`                                                                                                                             |
+| `name`       | [Customer's](#customer) name. Returned only if set.                                                                                    |
+| `email`      | [Customer's](#customer) email. Returned only if set.                                                                                   |
+| `avatar`     | [Customer's](#customer) avatar. Returned only if set.                                                                                  |
+| `created_at` | When [customer's](#customer) identity was created.                                                                                     |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.2/index.mdx]:
```suggestion
| `created_at` | Specifies when the [customer's](#customer) identity was created.                                                                                     |
```

**Pull request merged**